test-tools/issuer-front-end/src/Main.tsx

```diff
@@ -121,8 +128,13 @@ export default function Main(props: WalletConnectionProps) {
     const [browserPublicKey, setBrowserPublicKey] = useState('');
 
     const [issuerMetaData, setIssuerMetaData] = useState(EXAMPLE_ISSUER_METADATA);
+    const [updatedIssuerMetaData, setUpdatedIssuerMetaData] = useState('');
```

Comment:
Is there a reason you use `''` as "empty" rather than `undefined`?

Reply:
No particular reason except that most other `useState` hooks do the same on that front end. So either we keep it or should update the other `useState` hooks to be consistent.